external/llvm-project/mlir/include/mlir/Target/LLVM/ROCDL/Utils.h

```diff
@@ -31,6 +31,15 @@ StringRef getROCMPath();
 /// strings. By default this class serializes into LLVM bitcode.
 class SerializeGPUModuleBase : public LLVM::ModuleToObject {
 public:
+  /// AMDGCN standard libraries.
+  typedef enum : uint32_t {
```

Comment:
Could be an `enum class`?

Reply:
Sure, I can make it an enum class, the reason I didn't make it an enum class is because it's already scoped inside the `SerializeGPUModuleBase`, but I'll take it out of the class and make it an enum class.

Reply:
... Good point, it probably doesn't need to be an enum class because of the scoping.

I forget what upstream style on this sort of thing is